applications.json

```diff
@@ -1,5 +1,21 @@
 {
   "applications" : [
+    {
+      "short_description" : "cross platform e-book manager.",
+      "categories" : [
+        "Utilities"
+      ],
+      "repo_url" : "https://github.com/kovidgoyal/calibre",
+      "title" : "calibre",
+      "icon_url" : "",
+      "screenshots" : [
+        "https://github.com/kovidgoyal/calibre/blob/master/resources/images/lt.png"
+      ],
+      "official_site" : "",
```

Comment:
Please add "official_site" =https://calibre-ebook.com .

Reply:
do it